api/protobuf-spec/github_com_openshift_origin_pkg_build_apis_build_v1.proto

```diff
@@ -326,6 +326,12 @@ message BuildRequest {
 
   // DockerStrategyOptions contains additional docker-strategy specific options for the build
   optional DockerStrategyOptions dockerStrategyOptions = 9;
+
+  // incremental overrides the source-strategy incremental option in the build config
+  optional bool incremental = 10;
```

Comment:
If this is part of source strategy, why is this not part of a new SourceStrategyOptions struct?

Reply:
Updated